### PR TITLE
feat: add list-to-paragraph command

### PR DESCRIPTION
**Pull request by Saul-Mirone**

### CODE REVIEW

**Comment by Saul-Mirone** [comment]
* **#7962** <a href="https://app.graphite.dev/github/pr/toeverything/blocksuite/7962?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @Saul-Mirone and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: packages/blocks/src/list-block/commands/index.ts [hunk 1]
```diff
@@ -2,12 +2,22 @@ import type { BlockCommands } from '@blocksuite/block-std';
...
   unindentList: unindentListCommand,
 };
+
+declare global {
+  namespace BlockSuite {
+    interface CommandContext {
+      listConvertedId?: string;
+    }
+  }
+}
```
Comment (akumatus) [inline packages/blocks/src/list-block/commands/index.ts:23]: <i class='graphite__hidden'>[Re: line +24]<br/><br/></i>

Move this declare to file `list-to-paragraph.ts` to make it more cohesive？<span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/toeverything/blocksuite/7962?utm_source=unchanged-line-comment">Graphite</a>.</span>

**Review by akumatus — APPROVED** [review]

File: packages/blocks/src/list-block/commands/index.ts (hunk 1 above)
Comment (Saul-Mirone) [reply packages/blocks/src/list-block/commands/index.ts:23]: The `convertToNumberedList` command and `listToParagraph` command share the same output, what's the best practice here?

Comment (akumatus) [reply packages/blocks/src/list-block/commands/index.ts:23]: It is difficult to decide whether to declare it repeatedly in two files or declare it once in a shared file. I might prefer the former.

**Pull request merged**